Dockerfile

```diff
@@ -1,21 +1,30 @@
-# vim:set ft=dockerfile:
-FROM python:3.8-slim
-MAINTAINER https://github.com/pacificclimate/osprey
-LABEL Description="osprey WPS" Vendor="pacificclimate" Version="0.1.0"
```

Comment:
Why do you remove these?

Reply:
The first `FROM` statement is for the intermediate image, which is a side artifact and never used for any containers. The `MAINTAINER` and 'LABEL' statements are there to provide containers' metadata, thus they don't have to be in the intermediate image.